docker-compose.yml

```
@@ -47,6 +47,7 @@ services:
     volumes:
       - ./data/db:/var/lib/postgresql/data
       - ./_container_scripts/keyper-db-init.sh:/docker-entrypoint-initdb.d/keyper-db-init.sh:ro
+      - ./data/db-data:/var/lib/postgresql/dump
```

```suggestion
      - ./data/db-dump:/var/lib/postgresql/dump
```

`db-data` isn't a very self explanatory name.